cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -622,3 +621,58 @@ components:
         type: genetic_ancestry_value
       genetic_ancestry_South_Asian:
         type: genetic_ancestry_value
+      cell_line_ontology_term_id:
+        type: curie
+        dependencies:
+          - # If cell_type_ontology_term_id is CL:0000010 or CL:0000001
+            rule:
+              column: cell_type_ontology_term_id
+              match_exact:
+                terms:
+                  - CL:0000010
+                  - CL:0000001
+            error_message_suffix: >-
+              When 'CL:0000010”' or 'CL:0000001' for cell_type_ontology_term_id is True, this MUST be a child of term 'CL:0000010' within EFO. Otherwise this key MUST be 'na'.
+            type: curie
+            curie_constraints:
+              ontologies:
+                - EFO
+              allowed:
+                ancestors:
+                  EFO:
+                    - CL:0000010
+        curie_constraints:
+          ontologies:
+            - NA
+          exceptions:
+            - na
+        add_labels:
```

Comment:
here's how `add_labels` is supported for this field, roughly speaking:
- curators are required to pass in the column `cell_line_ontology_term_id`, which must be an ontology term following the defined constraints above
- given the term specified `cell_line_ontology_term_id`, map this ontology term to a human-readable term on the column `cell_line`
- the provided term in `cell_line_ontology_term_id` must follow the curie constraints defined within `add_labels`. so it must be within the EFO ontology graph